src/components/button/button.scss

```diff
@@ -187,15 +187,37 @@ button.md-button.md-fab {
   }
 }
 
-.md-toast-open-top {
-  .md-button.md-fab-top-left,
-  .md-button.md-fab-top-right {
-    transition: $swift-ease-out;
-    transform: translate3d(0, $button-fab-toast-offset, 0);
-    &:not([disabled]) {
-      &.md-focused,
-      &:hover {
-        transform: translate3d(0, $button-fab-toast-offset - 1, 0);
+// On small devices, the toast always appears on the bottom even if it should be on the top,
+// so don't move top FABs on small devices.
+@media (min-width: $layout-breakpoint-sm) {
```

Comment:
This `@media` check is redundant since what you're trying to achieve is binary (one or the other). For example, it's like changing `if/elseif/else` to `if/else`